dsweb/src/main/java/com/doctusoft/dsw/client/comp/TabSheet.java

```diff
@@ -3,48 +3,61 @@
 import com.doctusoft.bean.ValueChangeListener;
 import com.doctusoft.bean.binding.Bindings;
 import com.doctusoft.bean.binding.EmptyEventHandler;
+import com.doctusoft.bean.binding.ParametricEventHandler;
+import com.doctusoft.bean.binding.ValueBinding;
 import com.doctusoft.dsw.client.comp.model.TabSheetModel;
 import com.doctusoft.dsw.client.comp.model.TabSheetModel_;
 import com.google.common.base.Preconditions;
 
 public class TabSheet extends BaseComponent<TabSheet, TabSheetModel>{
 
+	private EmptyEventHandler defaultEventHandler;
+
 	public TabSheet() {
 		super(new TabSheetModel());
+
+		Bindings.obs(model).get(TabSheetModel_._activeTab).addValueChangeListener(new ValueChangeListener<Integer>() {
+			@Override
+			public void valueChanged(final Integer newValue) {
+				if (model.getAutomaticTabSwitch().equals(Boolean.TRUE)) {
```

Comment:
@mterjeki this isn't really more safe than just ```model.getAutomaticTabSwitch()``` - if it's null, your solution will still throw an NPE. The standard NPE-safe solution is ```Objects.firstNonNull(model.get..., false)```